client/coral-embed-stream/src/Embed.js

```diff
@@ -144,7 +158,10 @@ class Embed extends Component {
       <div style={expandForLogin}>
         <div className="commentStream">
           <TabBar onChange={this.changeTab} activeTab={activeTab}>
-            <Tab><Count count={asset.totalCommentCount}/></Tab>
+            <Tab><Count
+              count={asset.totalCommentCount}
+              handleClick={this.handleClick}/>
```

Comment:
Also we already have  `changeTab` for tab changing logic and we shouldn't spread these into different callbacks.